dotcom-rendering/scripts/webpack/webpack.config.browser.js

```diff
@@ -69,25 +60,15 @@ module.exports = ({ isLegacyJS, sessionId }) => ({
 						options: {
 							presets: [
 								'@babel/preset-react',
-								isLegacyJS
-									? [
-											'@babel/preset-env',
-											{
-												targets: {
-													ie: '11',
-												},
-												modules: false,
-											},
-									  ]
-									: [
-											'@babel/preset-env',
-											{
-												bugfixes: true,
-												targets: {
-													esmodules: true,
-												},
-											},
-									  ],
+								[
+									'@babel/preset-env',
+									{
+										bugfixes: true,
+										targets: {
+											esmodules: true,
```

Comment:
how you feel about using https://www.npmjs.com/package/@guardian/browserslist-config here?